elastic/elastic-agent
# Fix variable names in spec files

Some spec files have `preventions` fields referencing the nonexistent variable `runtime.user.root`. The correct variable name is `user.root`; presumably this was a typo that got copy-pasted between them. These preventions fields currently have no effect. This PR updates them with the correct name.

## Checklist


## How to test this PR locally

Run Agent with the relevant components, with and without root; before this PR it doesn't produce the configured error message, and afterwards it does. Alternatively (especially on a platform like Mac that requires root to even start), add the negated condition `${runtime.user.root} == true` to any component, and observe that the component still starts without a problem, whereas `${user.root} == true` correctly logs the configured error message.
